extensions/arc/runtime/src/main/java/io/quarkus/arc/properties/IfBuildProperty.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.arc.properties;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * When applied to a bean class or producer method (or field), the bean will only be enabled
+ * if the Quarkus build time property matches the provided value.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.METHOD, ElementType.TYPE, ElementType.FIELD })
+public @interface IfBuildProperty {
+
+    /**
+     * Name of the build time property to check
+     */
+    String name();
+
+    /**
+     * Expected {@code String} value of the build time property (specified by {@code name}) if the bean is to be enabled
+     */
+    String stringValue();
+
+    /**
+     * Determines if the bean is to enabled when the property name specified by {@code name} has not specified at all
```

Comment:
`bean is to enabled` -> `bean is to be enabled`
`has not specified ` -> `has not been specified`

Reply:
Fixed both